indico/modules/events/management/templates/clone_event.html

```diff
@@ -91,8 +91,13 @@
                             {% set count %}
                                 <span class="count"></span>
                             {% endset %}
-                            {% trans link='<a class="event-list">'|safe, endlink="</a>"|safe -%}
-                                {{ count }} new events will be created. {{ link }}When?{{ endlink}}
+                            {% trans link='<a class="event-list">'|safe, endlink="</a>"|safe,
+                                last_day='<span class="last-day">'|safe, endlast_day="</span>"|safe -%}
```

Comment:
I'd align this with `link` and consistently use single quotes (for the end tags). `endlast_day` also looks a bit weird. Maybe better to go for `lastday` and `endlastday` or `end_last_day` and `end_link`?